example-usage/workspace/.devcontainer/devcontainer.json

```diff
@@ -18,6 +18,10 @@
 		// Optional - For tools that require SSH
 		"ghcr.io/devcontainers/features/sshd:1": { }
 	},
+	"overrideFeatureInstallOrder": [
```

Comment:
Does order actually matter for these examples? I don't think these examples are meant to be a comprehensive view of all possible devcontainer options. If order doesn't matter, I would recommend removing this from the example to keep it simpler. `overrideFeatureInstallOrder` is not a property that most people should ever need to use.

Reply:
I just wanted to add a test coverage, so thought of adding to current devcontainer.json. I'll create a separate one for `overrideFeatureInstallOrder` test. 👍

Reply:
These `example-usage` ones are not currently covered by tests, but are setup as an example for users to see how to use the CLI in different scenarios. They were recently added in https://github.com/devcontainers/cli/pull/107.

Reply:
I have added a lot of tests recently under [`./src/test/container-features`](https://github.com/devcontainers/cli/tree/main/src/test/container-features), including example projects that the tests will execute.  I suggest adding a test there instead